File: app/src/main/java/com/katyabaygin/menu/ApiService.java
package com.katyabaygin.menu;

import io.reactivex.rxjava3.core.Single;
import retrofit2.http.GET;

public interface ApiService {

    @GET("api/json/v1/1/filter.php?a=Alcoholic")
    Single<DrinkResponse> loadDrinks();
}

File: app/src/main/java/com/katyabaygin/menu/data/ApiService.kt
package com.katyabaygin.menu.data

import com.katyabaygin.menu.domain.Drink
import retrofit2.http.GET

interface ApiService {
    @GET("api/json/v1/1/filter.php?a=Alcoholic")
    suspend fun loadDrinks(): List<Drink>
}




